sim/scenarios/blackhole/README.md

```diff
@@ -0,0 +1,42 @@
+# Blackhole
+
+This scenario uses a bottleneck link like the [../simple-p2p](simple-p2p)
+scenario. The link can be configured to start dropping (blackholing) all
+packets for a certain duration, and unblock some time later. Optionally,
+this pattern can be repeated.
+
+This scenario can be used to test various features of QUIC:
+* Short durations (around 3 RTTs) can be used to test PTOs.
+* Slightly longer durations (> 3 RTT + max_ack_delay) can be used to test
```

Comment:
@janaiyengar Please carefully check these numbers.

Reply:
I would remove these numbers from the description, they're not necessary, and can change in the future.